### ISSUE TITLE
ALF: Complete Set-up [TIMEBOX: 1 day]

### ISSUE THREAD

**Issue by kjwoodsISIS** [kjwoodsISIS/IBEX#457]
As developer and support person, I want to complete the set up of ALF, so that we can install the new control system on it.

'''Acceptance Criteria'''
1. Software pre-requisites for new control system installed on ALF control server.
1. Check networking (for private Galil network)
1. Test.

'''Notes'''
1. Freddie (16/07/2014): EPICS is now installed on ALF, galils need configuring and testing as well as the goniometer GUI.

**Comment by kjwoodsISIS** [comment]
_Imported from trac issue 458.  Created by skn09965 on 2014-07-10T16:24:28, last modified: 2014-09-10T14:26:00_

**Comment by kjwoodsISIS** [comment]
_Trac comment by skn09965 on 2014-09-05 09:18:37:_

Report from Freddie & Gareth:

From: ICP [mailto:icp-bounces@svr2.nd.rl.ac.uk] On Behalf Of freddie.akeroyd@stfc.ac.uk
Sent: 15 August 2014 15:54
To: icp@svr2.nd.rl.ac.uk
Subject: [ICP] Tests on ALF

Gareth and I ran some tests on ALF this afternoon. Tracing various cables, it turns out that the side of the ALF blockhouse nearest the Galil is actually patched back to ALF, so it was relatively easy to put the Galil onto the DAE private network. The epics Galil IOCs started up Ok and we were able to move motors via the motor record PVs. We were also able to start up CSS and view a motor with the motor screen, and to begin a run from genie python. However:
•	The ALF GUI has the wrong PV prefix (NDXALF:spudulike: rather than IN:ALF:) but after correcting this it still did not connect to PV’s, I think it might be missing the CAJ localhost patch? 
•	The is a 3.5 offset on the ROT axis that we need to understand and then incorporate into EPICS
•	The GONIO:Z etc axis PVs were created but did not set motor positions

Russell said he would look into getting us some time next week on ALF, Winfried is interested in seeing it in operation

Ticket complete.  A new ticket will be raised to address follow-on work.
